packages/whosmysanta-backend/src/server.js

```diff
@@ -5,22 +5,21 @@ import {connectDatabase} from './data';
 
 const app = express();
 
-// TODO: add auth middleware as an extra security layer
-// eg. app.use(auth0middleware);
+// TODO: add authentication / authorization middleware
 
 app.use(
   '/',
   graphqlHTTP({
     schema,
+    // TODO: Make sure this really doesn't run in prod.
+    // It was running in now when we first deployed.
     graphiql: process.env.NODE_ENV !== 'production',
   }),
 );
 
 app.listen(4000, (err) => {
-  if (err) {
-    console.error(err); // eslint-disable-line no-console
-  } else {
-    console.log('GraphQL server started'); // eslint-disable-line no-console
-    connectDatabase();
-  }
+  if (err) throw new Error(err);
+  console.log('GraphQL server started'); // eslint-disable-line no-console
+  connectDatabase();
+  console.log('Database connected!'); // eslint-disable-line no-console
```

Comment:
Even better, I used the promise interface they have: http://mongoosejs.com/docs/connections.html#callback